File: 448. Find All Numbers Disappeared in an Array/solution.py
class Solution:
    def findDisappearedNumbers(self, nums: List[int]) -> List[int]:
        rst = []
        setNum = set(nums)
        for i in range(1, len(nums) + 1):
            if i not in setNum:
                rst.append(i)
        return rst

File: 448. Find All Numbers Disappeared in an Array/solution.go
func findDisappearedNumbers(nums []int) []int {
	flags := make([]int, len(nums)+1)
	for _, num := range nums {
		flags[num]++
	}

	res := make([]int, 0)
	for i, flag := range flags {
		if i != 0 && flag == 0 {
			res = append(res, i)
		}
	}
	return res
}